pkg/cmd/server/api/types.go

```diff
@@ -20,6 +20,10 @@ var (
 	DeadKubernetesAPILevels    = []string{"v1beta1", "v1beta2"}
 	DeadOpenShiftAPILevels     = []string{"v1beta1"}
 
+	FeatureAliases = map[string][]string{
```

Comment:
> if you make this map aliases to their canonical feature, you get O(1) lookup and prevent duplicate aliases

Good idea. Will do.

> also, should you normalize the alias?

No. `NormalizeOpenShiftFeature()` returns canonical name for any give feature flag or its alias. I'll fix description.